index.html

```diff
@@ -52,18 +52,20 @@ <h1 class="page-header"><a href="https://ptpb.pw/" target="_blank">ptpb.pw</a></
               </button>
             </div>
             <div class="form-group">
-              <input type="file" id="file-input" class="sr-only">
-              <button id="file" type="submit" class="btn btn-default btn-block" disabled="disabled">
-                Upload File
+              <button id="shorturl" type="submit" class="btn btn-default btn-block" formaction="/u">
+                Short URL
                 <span class="fa fa-spin fa-spinner hidden"></span>
               </button>
             </div>
             <div class="form-group">
-              <button id="shorturl" type="submit" class="btn btn-default btn-block" formaction="/u">
-                Short URL
+              <button id="file" type="submit" class="btn btn-default btn-block">
```

Comment:
I suggest class=hidden, and unhide on `clear()`. If you did that, you might be able to get away with putting the file input here instead.